google/cloud/storage/bucket.py

```diff
@@ -440,7 +440,7 @@ def delete_blobs(self, blobs, on_error=None, client=None):
                     raise
 
     def copy_blob(self, blob, destination_bucket, new_name=None,
-                  client=None):
+                  client=None, preserve_acl=False):
```

Comment:
I see, I got that backwards.

What do you think about the alternate implementation that was posted in https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2389#issuecomment-249408321?

I like reducing the number of requests, but I think it's more DRY to do it the way you have outlined here.